### PR TITLE
Reworked AboutFragment

### PR DESCRIPTION
**Pull request by vanitasvitae**
I'd like to add an additional tab that displays the Changelog. Also I'd like to rework the LicenseFragment in the same style as the AboutFragment (if you like it gsantner)

### CODE REVIEW

**Comment by gsantner** [comment]
To avoid duplication: - Should be still read from the corresponding repository files:
- Contributors
- License

Everything else is "free play-room"

**Comment by vanitasvitae** [comment]
Of course. That actually came in mind, when I cleaned unused resources and stumbled over R.raw.changelog :D

Am 7. November 2016 16:29:52 MEZ schrieb Gregor Santner notifications@github.com:

> To avoid duplication: - Should be still read from the corresponding
> repository files:
> - Contributors
> - License
> 
> ## 
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/Diaspora-for-Android/diaspora-android/pull/115#issuecomment-258866776

## 

Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by gsantner** [comment]
Yeah, R.raw.changelog get dynamically loaded from CHANGELOG.md :D
(app/build.gradle  does that)

**Comment by vanitasvitae** [comment]
I know, that's great!

Am 7. November 2016 16:32:59 MEZ schrieb Gregor Santner notifications@github.com:

> R.raw.changelog get dynamically loaded from CHANGELOG.md :D
> 
> ## 
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/Diaspora-for-Android/diaspora-android/pull/115#issuecomment-258867692

## 

Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by vanitasvitae** [comment]
This can now be merged

File: app/src/main/java/com/github/dfa/diaspora_android/activity/AboutActivity.java [hunk 1]
```diff
@@ -169,12 +184,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                     e.printStackTrace();
                 }
             }
+            contributeBtn.setOnClickListener(this);
```
Comment (gsantner) [inline app/src/main/java/com/github/dfa/diaspora_android/activity/AboutActivity.java]: Could all be done by using

```
@OnClick( {R.id.id1, R.id.id2, R.id.id3} )
public void onSomethingClicked(View view){

}
```

Then also no Interface implementation is needed :D

And also: If nothing but the onClick event is needed, the `@BindViews` can even be removed :D

File: app/src/main/java/com/github/dfa/diaspora_android/activity/AboutActivity.java [hunk 2]
```diff
@@ -218,15 +279,27 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+                    Intent openBrowserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.fragment_license__license_gpl_link)));
+                    openBrowserIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                    startActivity(openBrowserIntent);
+                }
+            });
+
+            leafpicBtn.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    Intent openBrowserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.fragment_licesen__misc_leafpic_link)));
```
Comment (gsantner) [inline app/src/main/java/com/github/dfa/diaspora_android/activity/AboutActivity.java]: Call  `openLink()` Method? Maybe move over the openLink method to Helpers?

File: app/src/main/res/values-de/strings-about.xml [hunk 3]
```diff
@@ -11,30 +11,12 @@
   <string name="fragment_debug__section_log">Debug-Protokoll</string>
   <string name="fragment_debug__section_log_spam">Debug-Protokoll (Verbose)</string>
   <string name="fragment_debug__app_version">App Version: %1$s</string>
-  <string name="fragment_debug__package_name">Paketname: %1$s</string>
-  <string name="fragment_debug__android_version">Android Version: %1$s</string>
+    <string name="fragment_debug__android_version">Android Version: %1$s</string>
```
Comment (gsantner) [inline app/src/main/res/values-de/strings-about.xml]: Tabbing error?

**Comment by vanitasvitae** [comment]
I fixed the points you mentioned

**Pull request merged**